src/generic.rs

```diff
@@ -88,6 +98,18 @@ where
         }
     }
 
+    /// Extracts borrowed data.
+    ///
+    /// Panics: If the Beef is owned.
+    #[inline]
+    pub fn unwrap_borrowed(self) -> &'a T {
+        let cow = ManuallyDrop::new(self);
```

Comment:
Added tests in #21, this looked like a leak to me, Miri agreed :).

Reply:
Sorry about that!